### DISCUSSION TITLE
"Append File" to  offset?

### DISCUSSION THREAD

**Discussion by LsrSal** [arkypita/LaserGRBL#1931] [Ideas]
Would be nice to be able to offset appended object to the work. Presently all referenced to absolute origin. Most of the time all objects rendered close to origin and adding them creates pileup mess.
Ideally I would like to be able to drag appended object to visually align it before proceeding.

Simple but practical **solution can be to prompt to add  x:y offset before rendering appended object.**

**Comment by StuartB4** [comment]
You can already do that with the X and Y offset settings in Target image settings.
Load your first object, click Append and load the second one, then adjust X and Y offset numbers to position it where you want.

**Reply by LsrSal** [reply]
Right, but for the image objects only.
I have prepared NC. When appending that - no option for the offset of NC files.
